client/my-sites/checkout/cart/secondary-cart.jsx

```diff
@@ -3,7 +3,7 @@
 /**
  * External dependencies
  */
-
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
```

Comment:
Nit: This likely should be below `PropTypes` / `React`. Ideally, imports should be alphabetized to improve readability. Not a big deal though.

Reply:
@tyxla Thanks for that check! Can you quickly confirm this is what you meant? https://github.com/Automattic/wp-calypso/pull/28910/commits/e61e341b5f03aefde7a56265911f226dcae6226a

Reply:
Yeah, that's right @keoshi!  👍 

The imports around seem to be not sorted too, but you don't have to sort them here, as they were like that before already 😉

Reply:
Thanks! 🙌